File: src/sbt-test/lagom-bundle-plugin/multi-projects-same-service-name/credit-api/src/main/java/api/CreditService.java
package api;

import akka.stream.javadsl.Source;

import akka.NotUsed;
import com.lightbend.lagom.javadsl.api.ServiceCall;
import com.lightbend.lagom.javadsl.api.Descriptor;
import com.lightbend.lagom.javadsl.api.Service;
import com.lightbend.lagom.javadsl.api.transport.Method;
import static com.lightbend.lagom.javadsl.api.Service.*;

public interface CreditService extends Service {

  ServiceCall<NotUsed, NotUsed> credit();

  @Override
  default Descriptor descriptor() {
    return named("creditservice").withCalls(
      restCall(Method.GET,  "/credit", this::credit)
    ).withAutoAcl(true);
  }
}

File: src/sbt-test/lagom-bundle-plugin/multi-services-one-project-same-service-name-scala/payment-api/src/main/scala/api/CreditService.scala
package api

import akka.NotUsed
import com.lightbend.lagom.scaladsl.api._

trait CreditService extends Service {

  def credit: ServiceCall[NotUsed, NotUsed]

  override def descriptor: Descriptor = {
    import Service._
    named("paymentservice")
      .withCalls(call(credit))
      .withAutoAcl(true)
  }
}